site/content/help/feature/custom-textures/index.md

```diff
@@ -3,18 +3,54 @@ title = "Custom Textures"
+
+## Texture dumping
+
+Citra will dump textures when they are refereced by the game and uploaded from the guest VRAM to the host memory. Dumped textures may only have power-of-two dimensions in order to avoid dumping host framebuffers.
+
+The texture dumper will write a template **`pack.json`** file in the dump directory, which should be copied in the load folder when testing the texture pack. By default textures will be dumped using the new hashing method. If a texture pack exists in the load folder the dumper will use the same hashing method as that pack. Regardless of the hashing method, packs should still load correctly on any recent build of Citra. For any concerns regarding pack compatibility, possible breakages or feature requests please contact us either on [Github](https://github.com/citra-emu/citra/issues/) or [Discord](https://citra-emu.org/discord/), as this is still a very new feature.
+
+The filename of dumped textures is comprised of various information about the guest texture. For example the filename **tex1_256x256_543624189C94B105_12_mip0.png** includes the guest texture dimensions **256x256**, the texture hash **543624189C94B105**, the guest pixel format **12** and the mipmap level of the texture **mip0**
+
+## Pack configuration
+
+The custom textures rework introduced a new json configuration file for texture packs, which is used to distinguish between old and new packs. It contains information about the pack such as name, author, as well as other configuration options like the used hash format, automatic mipmap generation and control for texture flipping. The latter two options are **unsupported** when compressed texture formats are used.
+
+The new hashing option is recomended for new texture packs and is enabled by default. Advantages include faster hashing, as it requires less input data compared to the old hash and better compatibility between graphics APIs. In order to maintain compatibility with existing packs, loading the older hashing format is still fully supported. Usage of the old hash occurs either when the pack.json file does not exist in the texture pack load folder, or the use_new_hash option is disabled.
+
+The configuration file also allows for hash mappings, which means textures can have arbitrary filenames that don't strictly adhere to the dumper naming guidelines. For example:
+
+```
+"textures" : {
+  "114BFC385ED72F15" : "logo.png"
+  "22B8C43233F640AE" : "sky.png"
+}
+```
+
+## Normal maps
+
+The new custom texture system supports custom normal maps. This feature allows creators to add additional details to objects that would not be possible with only diffuse maps.
+
+Normal maps have the same filename as their diffuse counterpart, with an added **`.norm`** prefix. For example the normal map for dumped texture **texture.png** will be named **texture.norm.png**. This applies for all supported file types and for hash mappings.
```

Comment:
"prefix *before the file extension*".
or
"`.norm.` between filename and file extension." (to clarify that foobar.norm.png works for foobar.dds)